src/registrar/models/domain.py

```diff
@@ -954,19 +1139,26 @@ def _fetch_cache(self, fetch_hosts=False, fetch_contacts=False):
                 "tr_date": getattr(data, "tr_date", ...),
                 "up_date": getattr(data, "up_date", ...),
             }
-
             # remove null properties (to distinguish between "a value of None" and null)
             cleaned = {k: v for k, v in cache.items() if v is not ...}
 
             # statuses can just be a list no need to keep the epp object
             if "statuses" in cleaned.keys():
                 cleaned["statuses"] = [status.state for status in cleaned["statuses"]]
-            # get contact info, if there are any
+
+            # Registrant should be of type PublicContact
+            if "registrant" in cleaned.keys():
+                # For linter...
+                _ = cleaned["registrant"]
```

Comment:
This is interesting, what error did linter throw? Can you instead do self._registrant_to_public_contact(cleaned["registrant"])

Reply:
Just a line length error, I think I may just rename the function to something shorter... Its an item for code cleanup